### PR TITLE
responsiveness

### PR DESCRIPTION
**Pull request by preethasri**
added responsiveness to video cards please do preview

### CODE REVIEW

**Review by Chakravarthi0 — APPROVED** [review]
code looks clean.
suggestion: you can add some gap to categories filter
image

File: src/Pages/Home/Components/VideoCard.css [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
     color:white;
 }
 .video-info{
-    display:flex;
+
     background-color:var(--primary-color);
-    flex-direction: column;
-    color:white;
+    height: max-content;
     width:300px;
```
Comment (bipin7yadav) [inline src/Pages/Home/Components/VideoCard.css:19]: you can rem instead of px

**Review by bipin7yadav — APPROVED** [review]

File: src/Pages/Home/Components/VideoCard.css (hunk 1 above)
Comment (preethasri) [reply src/Pages/Home/Components/VideoCard.css:19]: will work on that bipin

**Comment by preethasri** [comment]
> code looks clean. suggestion: you can add some gap to categories filter image

will work on that saran

**Pull request merged**